app/src/main/res/drawable/ic_baseline_person_24.xml

```diff
@@ -1,5 +1,5 @@
 <vector android:autoMirrored="true" android:height="24dp"
-    android:tint="?attr/colorControlNormal" android:viewportHeight="24"
+    android:tint="?attr/material_theme_secondary_color" android:viewportHeight="24"
```

Comment:
If this drawable will be tinted explicitly wherever it's used, does it need its own tint here?

Reply:
Correct. removed @dbrant